django/db/models/base.py

```diff
@@ -1708,7 +1708,10 @@ def _check_ordering(cls):
             fld = None
             for part in field.split(LOOKUP_SEP):
                 try:
-                    fld = _cls._meta.get_field(part)
+                    if part == 'pk':
```

Comment:
I think we should check also previous field i.e.
```python
    if part == 'pk' and not fld == _cls._meta.pk
```
because without this we will allow for e.g. `parent__pk__pk`.

Reply:
Thanks. Done. Do we need to add a test for `parent__pk__pk`?